packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -64,7 +64,7 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
       cacheKey: ''
     };
 
-    this._classNames = getClassNames(styles!, { theme: theme!, className });
+    this._classNames = getClassNames(styles!, { theme: theme! });
```

Comment:
@micahgodbolt I'm a bit concerned if this is the right chang. I'm not sure why the `root` style is not applied to the `ResizeGroup` actual root element vs. the `FocusZone` which is a child of the `ResizeGroup`. 

This change applies `className` prop to the actual `root` element. I just want to confirm: (1) this is the right dupe to remove and (2) `root` style is applied to the correct element. Thanks

Reply:
className should be on the root, not inside.

Reply:
agreed, classname should always be the root visual div (I say visual as layered things have their root inside of the layer)

Reply:
So you're both confirming `styles.root` is applied at the right level as it currently is, correct?

Reply:
Please note `styles.root` IS applied inside and I want to make sure we're all on the same page here.

Reply:
![image](https://user-images.githubusercontent.com/26070760/47451262-287d0a80-d77c-11e8-98dd-ac409cf8d208.png)